staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2874,6 +2874,229 @@ metadata:
 `, string(updatedCRYaml))
 }
 
+func TestApplyWithPruneV2ApplySetNamespace(t *testing.T) {
```

Comment:
Could we add this to TestApplyWithPruneV2 ?  It's hard to tell how similar it is / is not :-)